epoxy-composesample/build.gradle

```diff
@@ -0,0 +1,69 @@
+plugins {
+  id 'com.android.application'
+  id 'kotlin-android'
+}
+apply plugin: 'kotlin-kapt'
+
+android {
+  compileSdkVersion 30
+  buildToolsVersion "30.0.3"
+
+  defaultConfig {
+    applicationId "com.airbnb.epoxy.compose.sample"
+    minSdkVersion 21
+    targetSdkVersion 30
+    versionCode 1
+    versionName "1.0"
+
+    testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+  }
+
+  buildTypes {
+    release {
+      minifyEnabled false
+      proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+    }
+  }
+  compileOptions {
+    sourceCompatibility JavaVersion.VERSION_1_8
+    targetCompatibility JavaVersion.VERSION_1_8
+  }
+  kotlinOptions {
+    jvmTarget = '1.8'
+    useIR = true
+  }
+  buildFeatures {
+    compose true
+  }
+  composeOptions {
+    kotlinCompilerExtensionVersion COMPOSE_VERSION
+  }
+}
+
+kapt {
+  correctErrorTypes = true
```

Comment:
I don't think this is needed?

Reply:
It says in the epoxy doc that if I'm using Kotlin I need to use this

Reply:
Yeah, that's only for AutoModels which is legacy, I just haven't kept the readme up to date